### ISSUE TITLE
Webpage not loading - Android

### ISSUE THREAD

**Issue by arundev-kv** [react-native-webview/react-native-webview#1530]
Getting the error

`{
canGoBack: false,
canGoForward: false,
code: 3,
description: "SSL error: The certificate authority is not trusted",
loading: false,
target: 4455,
title: "",
url: "https://myurl"
}`
 in `onError` event.

        "react-native": "0.61.5",
        "react-native-webview": "^10.3.2",

**Comment by Unity-Group-BFGI** [comment]
same here,i when i include uri in my webview using native-webview, page was like blank, then i open it as web, and i inspect, there was no div or something which holds the webview content, it was like <div> </div> empty..
solve this issue please

**Comment by Stefanama** [comment]
same thing happens here - tested locally and appears to be solved by https://github.com/react-native-community/react-native-webview/pull/1466
@thephpjedi any idea when that fix could go out? 🙏

**Comment by thephpjedi** [comment]
@Stefanama No idea, up to maintainers. I pinged again in #1466 to see if it can get merged.

**Comment by Stefanama** [comment]
thanks a lot @thephpjedi! 😄 
yep, that's fine, unfortunately that resource is not under our control but also don't need it on the app's webview (intercom in this case, which sounds weird, makes me think issue might be something else). Your fix works perfectly for what we need 💪

**Comment by alexpchin** [comment]
I'm having an issue with Android truncating the `www` from my urls? Is this due to: https://www.techrepublic.com/article/google-finally-chops-www-from-chromes-address-bar-despite-backlash-over-confusing-change/ 

Is react-native-webview chopping off www because the default browser is Chrome?

**Comment by thephpjedi** [comment]
@alexpchin This issue is unrelated to your question. Please open a new issue.

**Comment by alexpchin** [comment]
@thephpjedi Are you sure that your SSL is a wildcard and covers www. and no www? This is why you might be getting an error.

**Comment by thephpjedi** [comment]
@alexpchin: I think you meant to send the comment to someone else. I'm not OP.

**Comment by Unity-Group-BFGI** [comment]
I'm Stuck in between my project, and only 2 days are remaining for its deadline ends, first i made by websites on wordpress that is quite slow and i plug webview using andriod studio using java, it works but tooooo slowly :( and also when i rotate, it got app is stopped working, when i minimize or maximize, ok fine, then i use cordova and i plug iframe, quite good but x-frame CSP policy warnings, then i used kordular, they work same as other, all was good, but when i pressed on external link they just carried aways, like playstore, amazon app, zoom app, url to inside webview, i want them to open - a open with option, but nothing solved, at the end, i m good with react so i started react native, but wth these WEBSITE LOADING ERROR ???? :''( 
help me jesusssss

**Comment by Stefanama** [comment]
@Unity-Group-BFGI not sure about your internal practices but as you're on a tight deadline, first thing I'd suggest is to apply that fix I linked locally and see if that solves your problem otherwise you risk waiting on a fix that will be useless for your use case.

**Comment by Unity-Group-BFGI** [comment]
25k is very much for me in india, i will not get 25k rupee :(, well i find onemore thing use full i m using flutter now, and i created , its works fast then android studio java, and i made backbutton, forward button, reload button and with there animations :D , so maybe my problem will solve by morning, and for external apps links, i m using a swatch button on my right and embading a direct link and open it as external broswer :P

**Comment by sujayjaju** [comment]
Has anyone figured this error?
The URL opens ok in the browser - but in webview throws this error for:
https://fonts.googleapis.com/css?family....

**Comment by willcaul** [comment]
@Titozzz @jamonholmgren - any chance @alesandroortiz's fix could be looked at? I'm getting SSL cert errors from a 3rd-party tracking pixel loaded on my page.

**Comment by Dror-Bar** [comment]
I'm still getting this ssl error, and I'm using latest version (11.6.4).

**Comment by vickypathak123** [comment]
Any Solution ?

**Comment by yunau** [comment]
I have the same issue and I'm using the latest version (11.6.4) too.
The ios version is fine but the Andriod version is running with the SSL error.
No luck when adding "ignoreSslError={true}" on WebView, and got a blank page.

**Comment by koaladlt** [comment]
Same issue here, on Pixel 4 emulator

**Comment by parasjain10** [comment]
![image](https://github.com/user-attachments/assets/3c358197-d141-4e10-9d99-b8f84854c9d9)

I am getting this issue

React Native version: 0.76.1
React Native Web view: 13.12.4
